core/client/queries/get-products.ts

```diff
@@ -0,0 +1,219 @@
+'use server';
```

Comment:
We're using `'use server'` for files that don't actually have server actions. These are being used in route handlers, there's no need for `'use server'` in these files